src/web/lib/braze/parseBrazeEpicParams.test.ts

```diff
@@ -127,6 +136,22 @@ describe('parseBrazeEpicParams', () => {
 	});
 
 	it('returns null when the buttonUrl is missing', () => {
+		const dataFromBraze: EpicDataFromBraze = {
+			componentName: 'Epic',
+			heading: 'Example Heading',
+			paragraph1: 'Paragraph 1',
+			paragraph2: 'Paragraph 2',
+			highlightedText: 'Example highlighted text',
+			buttonText: 'Button',
+			ophanComponentId: 'epic_123',
+		};
+
+		const got = parseBrazeEpicParams(dataFromBraze);
+
+		expect(got).toBeNull();
+	});
+
+	it('returns null when the trackingId is missing', () => {
```

Comment:
It looks like this test has two missing fields (`buttonUrl` and `ophanComponentId`). Should this be two separate tests?

Reply:
Good spot, sorted now.